packages/storybook/src/nimble/breadcrumb/breadcrumb.mdx

```diff
@@ -12,15 +14,27 @@ See the [nimble-angular Readme](https://github.com/ni/nimble/blob/main/packages/
 for information on using this component in Angular with RouterLink directives.
 
 <Canvas of={breadcrumbStories._standardBreadcrumb} />
+
+## API
+
 <Controls of={breadcrumbStories._standardBreadcrumb} />
-<Stories of={breadcrumbStories} />
+<ComponentApisLink />
+
+### Breadcrumb Item
```

Comment:
This is following the pattern I've started for other pages that document multiple components, so I'll leave it as-is for now and address it more holistically. Captured in #824.

It's also felt weird to me that the "API" heading is in between the canvas and the args table that controls that canvas. But I do want a canvas to appear "above the fold" so that users see a working example right next to the description.

Maybe:
```
# Component Name
Description
Canvas

# API
## component-tag
Canvas
Args table

## child-component-tag
Canvas
Args table
```